src/net_processing.cpp

```diff
@@ -2813,6 +2816,12 @@ void PeerManagerImpl::ProcessGetData(CNode& pfrom, Peer& peer, const std::atomic
                 push = true;
             }
         }
+        if (!push && inv.type == MSG_DSQ) {
+            if (auto opt_dsq = m_cj_ctx->server->GetQueueFromHash(inv.hash); opt_dsq.has_value()) {
```

Comment:
same re queueman, smth like
```suggestion
            auto opt_dsq = m_cj_ctx->server->GetQueueFromHash(inv.hash);
            if (!opt_dsq.has_value()) {
                opt_dsq = m_cj_ctx->queueman->GetQueueFromHash(inv.hash);
            }
            if (opt_dsq.has_value()) {
```

Reply:
why / how could not server have it but queue man would?